plugins.json

```diff
@@ -2374,6 +2374,13 @@
             "summary": "A tree-sitter grammar for nu-lang, the language of nushell",
             "license": "MIT",
             "rockspec_template": "nurr/resources/tree-sitter-nu.template"
+        },
+        {
+            "name": "neanias/everforest-nvim",
+            "shorthand": "everforest-nvim",
+            "dependencies": [],
+            "summary": "A Lua port of the Everforest colour scheme",
+            "license": "No License"
```

Comment:
note: I would suggest requesting that the plugin author add a license.
According to the [GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license):

> You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work.

Reply:
Done !